applications/mlflow/DEVELOPMENT.md

```diff
@@ -2,42 +2,106 @@
 
 This document provides information about developing, testing, and releasing the MLflow application using the included Taskfile.
 
+## Development
+
+## Prerequisites
+
+- Docker
```

Comment:
Let's decouple from requiring Docker itself and make it any compatible container orchestrator. Podman is quickly becoming the standard in linux distributions with Docker often being shipped as an external repo.